src/components/projects/create-wp-form/create-wp-form/create-wp-form.tsx

```diff
@@ -40,34 +43,72 @@ const CreateWPFormView: React.FC<CreateWPFormViewProps> = ({
               <Col>
                 <Row>
                   <Form.Group as={Col}>
-                    <Form.Label htmlFor='wp-name'>Work Package Name</Form.Label>
-                    <Form.Control id='wp-name' name='wpName' type='text' required></Form.Control>
+                    <Form.Label htmlFor="wp-name">Work Package Name</Form.Label>
+                    <Form.Control
+                      id="wp-name"
+                      name="name"
+                      type="text"
+                      onChange={(e) => name(e.target.value)}
+                      required
+                    ></Form.Control>
                   </Form.Group>
                 </Row>
                 <Row>
                   <Form.Group as={Col}>
-                    <Form.Label htmlFor='project-wbs-num'>Project WBS Number</Form.Label>
-                    <Form.Control id='project-wbs-num' name='wbsNum' type='text' required></Form.Control>
+                    <Form.Label htmlFor="project-wbs-num">Project WBS Number</Form.Label>
+                    <Form.Control
+                      id="project-wbs-num"
+                      name="wbsNum"
+                      type="text"
+                      onChange={(e) => wbsNum(e.target.value)}
+                      required
+                    ></Form.Control>
+                  </Form.Group>
+                  <Form.Group as={Col}>
+                    <Form.Label htmlFor="crId">Change Request ID</Form.Label>
+                    <Form.Control
+                      id="crId"
+                      name="crId"
+                      type="number"
+                      min={1}
+                      onChange={(e) => crId(parseInt(e.target.value))}
+                      required
+                    ></Form.Control>
                   </Form.Group>
                 </Row>
                 <Row>
                   <Form.Group as={Col}>
-                    <Form.Label htmlFor='start-date'>Start Date</Form.Label>
-                    <Form.Control id='start-date' name='startDate' aria-label={'start date input'} type='date' required></Form.Control>
+                    <Form.Label htmlFor="start-date">Start Date</Form.Label>
```

Comment:
```suggestion
                    <Form.Label htmlFor="start-date">Start Date (YYYY-MM-DD)</Form.Label>
```

Reply:
this is because it will error with different formats that people might think of using